services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -44,6 +71,7 @@ export default class ApplyPage extends React.Component<Props> {
               phone: '',
               email: '',
               confirmEmail: '',
+              commissionIds: [''] as string[],
```

Comment:
You should be able to replace `['']` with `['']`. Otherwise the `max` check in your validation will think there’s an extra commission checked.